packages/corpus/lib/report.dart

```diff
@@ -58,11 +76,11 @@ class Report {
     buf.writeln();
     buf.writeln('## General info');
     buf.writeln();
-    if (reportTarget.isDartLibrary) {
+    if (reportTarget is DartLibrary) {
       buf.writeln('https://api.dart.dev/dart-${reportTarget.name}/'
           'dart-${reportTarget.name}-library.html');
     } else {
-      buf.writeln(reportTarget.description);
+      buf.writeln((reportTarget as PackageTarget).description);
```

Comment:
If `description` were a nullable property of ReportTarget we could avoid this cast.

Reply:
Added the type check to clarify, but I would rather not make this a property of `ReportTarget` as a `DartLibraryTarget` really has no description (so far).